js/samples/04.ai.b.messageExtensions.AI-ME/src/index.ts

```diff
@@ -106,15 +106,15 @@ const promptManager = new DefaultPromptManager<ApplicationTurnState>(path.join(_
 // Define storage and application
 // - Note that we're not passing a prompt for our AI options as we won't be chatting with the app.
 const storage = new MemoryStorage();
-const app = new Application<ApplicationTurnState>({
-    storage,
-    adapter,
-    botAppId: process.env.MicrosoftAppId,
-    ai: {
+const botAppId = process.env.MicrosoftAppId as string;
```

Comment:
prefer assigning empty string over type casting
```suggestion
const botAppId = process.env.MicrosoftAppId || '';
```